apps/explorer/src/components/Table/SuiAmount.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
moved this into the `Table` directory and fixed up the styles. let me know if you had something else in mind for this @Jordan-Mysten - figured it would be useful to have a component for displaying amounts in tables